test-end-to-end/android-maestro/e2e-tests/002_004_course_title_is_mandatory.yaml

```diff
@@ -0,0 +1,30 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/002_004_course_title_is_mandatory"
+  TESTNAME: "002_004_course_title_is_mandatory"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+- runFlow: "../common/clear_state_and_get_started.yaml"
+- runFlow: "../common/select_site.yaml"
+- runFlow:
+    file: "../common/user_login.yaml"
+- runFlow:
+    file: "../common/add_course.yaml"
+    env:
+      TESTCOURSENAME: " "
+# assert to make sure the course is not saved because of blank course title
+- assertVisible: "SAVE"
```

Comment:
Yes the floating action button is not visible when editing, but I think it would be better to use the title for an assertion. Also we should make an assertion that the "required field" message appears.